CSharpSourceCode/HarmonyPatches/CampaignPatches.cs

```diff
@@ -85,14 +175,14 @@ public static void Postfix(ref Settlement __result, Kingdom __instance)
         //This behaviour contains hardcoded hero / lord references that are not present because we skip loading the vanilla files.
         [HarmonyPrefix]
         [HarmonyPatch(typeof(BackstoryCampaignBehavior), "RegisterEvents")]
-        public static bool Prefix3()
+        public static bool WhyIsThisNeeded()
```

Comment:
I think this method stops main character family from being created, maybe rename to something like  "ChangePlayersBackstory" we might need it later